astro/src/content/docs/release-notes/index.mdx

```diff
@@ -16,11 +16,46 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.50.0' releaseDate='April 24, 2024' />
+
+<GeneralMigrationWarning >
+  This release makes significant changes to the default behavior of new Applications with regard to scopes in OAuth workflows.
+  The database migration will update existing Applications to behave in a backwards compatible manner.
+  See the OAuth [Scopes](/docs/lifecycle/authenticate-users/oauth/scopes) documentation for more information, in particular the `Relationship`, `Unknown scope policy`, and `Scope handling policy` configurations.
+</GeneralMigrationWarning>
+
+<DatabaseMigrationWarning />
+
+### Changed
+* The `/oauth2/userinfo` endpoint now requires the `aud` claim to be present on the provided token. See the [UserInfo endpoint](/docs/lifecycle/authenticate-users/oauth/endpoints#userinfo) for more detail.
+
+  If you are not using OAuth, and your JWT does not contain the `aud` claim, consider using the [JWT validate](/docs/apis/jwt#validate-a-jwt) API instead
+* Applications now offer an Unknown scope policy configuration. This can be used to enhance security by rejected or removing unknown scopes during an OAuth workflow. See the application [Scopes tab](/docs/get-started/core-concepts/applications#scopes) for more detail.
```

Comment:
```suggestion
* Applications now offer an `Unknown` scope policy configuration. This can be used to enhance security by rejected or removing unknown scopes during an OAuth workflow. See the application [Scopes tab](/docs/get-started/core-concepts/applications#scopes) for more detail.
```